presto-orc/src/test/java/io/prestosql/orc/OrcTester.java

```diff
@@ -652,8 +653,8 @@ private static void writeOrcColumnPresto(File outputFile, Format format, Compres
         metadata.put("columns", "test");
         metadata.put("columns.types", createSettableStructObjectInspector("test", type).getTypeName());
 
-        OrcWriter writer;
-        writer = new OrcWriter(
+        @SuppressWarnings("resource")
```

Comment:
Can you make sure `writer` gets always closed? 
Perhaps `new FileOutputStream` should be wrapped in t-w-r individually

Reply:
I'm going to revert this `@SupressWarnings` and leave it alone for now. We have to use it after the `close()` to call the `validate()` method, and IntelliJ still warns about not using it with try-with-resources even if you do
```java
OrcWriter writer = ...;
try (Closeable ignored = writer) {
    ...
}
writer.validate(...);
```
It's test code so I'm not too concerned about a leak during a test failure (though it should be fixed at some point).

However, I did notice that `OrcWriter.close()` itself can leak if there is an error during close, which I will fix separately since that can leak resources.